CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Patch
 
+- Avatar: Add `__dangerouslyUseDefaultIcon` prop
```

Comment:
@nachofassini Can we make this `minor` instead since we are changing the API for `<Avatar />`?